.github/workflows/deploy_app.yml

```
@@ -0,0 +1,52 @@
+name: Deploy to Production
+
+on:
+  push:
+    branches:
+      - zooniverse-deployment
+  workflow_dispatch:
+
+jobs:
+  build_and_push_image:
+    name: Build and Push Image
+    uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@main
+    with:
+      repo_name: camera-traps-api
+      commit_id: ${{ github.sha }}
+      file: ./api/batch_processing/api_core/Dockerfile
+      latest: true
+
+  deploy_app:
+    runs-on: ubuntu-latest
+    needs: [build_and_push_image]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3.5.2
```

I updated to this version in the shared workflows, thanks for the reminder.
```suggestion
      uses: actions/checkout@v4.1.1
```